rtengine/procparams.h

```diff
@@ -1343,7 +1343,7 @@ struct DehazeParams {
     int strength;
     bool showDepthMap;
     int depth;
-
+    bool luminance;
```

Comment:
This killed a blank line which seperated data from functions.

Reply:
Changed